.buildkite/pipeline.yml

```diff
@@ -168,3 +168,30 @@ steps:
     plugins:
       - junit-annotate#v2.4.1:
           artifacts: build/TEST-go-integration*.xml
+
+  # Branch steps
+  - group: "K8s tests"
+    key: "k8s-tests"
+    steps:
+      - label: "K8s tests: {{matrix.k8s_version}}"
+        branches: "main 7.* 8.* v7.* v8.* buildkite_k8s_tests"
```

Comment:
the `buildkite_k8s_tests` should make it run on the source branch but I don't see the k8s tests running anywhere... Is this on purpose or am I misreading the intention here ?

Reply:
now we run it on every PR.